antora-playbook.yml

```diff
@@ -43,7 +43,6 @@ asciidoc:
     identity-provider-url: https://www.keycloak.org/
     identity-provider-version: "6.0"
     identity-provider: Keycloak
-    identity-provider2: Keycloak
```

Comment:
How will this get reflected in the downstream? Those are two district attributes there.

Reply:
I have an MR for downstream that fixes it.